geo_activity_playground/core/activity_parsers.py

```diff
@@ -120,13 +123,26 @@ def read_fit_activity(path: pathlib.Path, open) -> pd.DataFrame:
                             row["altitude"] = fields["altitude"]
                         if "enhanced_altitude" in fields:
                             row["altitude"] = fields["enhanced_altitude"]
+                        if "grade" in fields:
```

Comment:
Those three additional fields are now added to the time series. There is no analysis on them yet, though.